pkg/jobs/trigger_interval.go

```diff
@@ -0,0 +1,69 @@
+package jobs
+
+import (
+	"errors"
+	"time"
+
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// IntervalTrigger implements the @interval trigger. It triggers a job on a
+// fixed period of time.
+type IntervalTrigger struct {
+	du   time.Duration
+	in   *TriggerInfos
+	done chan struct{}
+}
+
+// NewIntervalTrigger returns a new instace of IntervalTriven given the
+// specified options.
+func NewIntervalTrigger(infos *TriggerInfos) (*IntervalTrigger, error) {
```

Comment:
We have the same thing for `@in`. If you add a trigger to send you a reminder by email in one month, and that the stack is restarted between, it will reset the counter.

Maybe we should transform `@in` triggers in `@at` triggers, and `@interval` triggers in `@cron`.